ts/features/wallet/onboarding/paypal/screen/__tests__/PayPalPpsSelectionScreen.test.tsx

```diff
@@ -25,26 +28,58 @@ describe("PayPalPpsSelectionScreen", () => {
   jest.useFakeTimers();
   const globalState = appReducer(undefined, applicationChangeState("active"));
   const store = createStore(appReducer, globalState as any);
-  it(`screen should be defined`, () => {
+  it(`screen should be defined when the psp list is ready`, () => {
     const render = renderComponent(store);
+    render.store.dispatch(searchPaypalPsp.success(pspList));
     expect(
       render.component.queryByTestId("PayPalPpsSelectionScreen")
     ).not.toBeNull();
   });
 
-  it(`footer buttons should be defined`, () => {
+  it(`footer buttons should be defined when the psp list is ready`, () => {
+    const store = createStore(appReducer, globalState as any);
```

Comment:
There is a specific reason why you redeclare the store in every test case?
What do you think of centralized it in the `renderComponent` function?

Reply:
Good point!
It could be refactored
done here fc561b468aab31d227c744b15b22867404cb0655